packages/react/src/patterns/blocks/LogoGrid/README.md

```diff
@@ -30,24 +30,34 @@ import ReactDOM from 'react-dom';
 import { LogoGrid } from '@carbon/ibmdotcom-react';
```

Comment:
@reesedownes you may need to rename this to have the .mdx extension for this to work, check the other mdx files as well on importing prop table, etc.

Reply:
@jeffchew fixed. the docs tab rendered locally.

also, i realized that the way i was doing the CTA props was wrong before. i had it before that you passed in an entire CTA object, but since the design intent was that the CTA is always a card with an arrow icon, i was exposing too many properties.

so instead i changed it to ctaCopy and ctaHref, and the entire CTA object is defined in LogoGrid itself. let me know if you think this is incorrect.